lib/engine_credo/cli.ex

```diff
@@ -17,7 +17,7 @@ defmodule EngineCredo.CLI do
   rescue
     error ->
       # credo:disable-for-next-line Credo.Check.Warning.IoInspect
-      IO.puts(:stderr, Exception.format(:error, error))
+      IO.puts(:stderr, Exception.format(:error, error, System.stacktrace()))
```

Comment:
Would it be possible to use `__STACKTRACE__` here instead? The `System.stacktrace/0` function has been deprecated.

Reply:
Sure! Nice catch :baseball:!

Done in 1df5157